src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -147,7 +148,6 @@ public function loadPublicAssets()
             'client-id' => $merchant->clientId,
             'merchant-id' => $merchant->merchantIdInPayPal,
             'components' => 'hosted-fields,buttons',
-            'locale' => get_locale(),
```

Comment:
@ravinderk Is there nothing to do on the JS side?

Reply:
@JasonTheAdams I tested it. We do not need this if we want PayPal to handle translation.